tests/pytests/unit/modules/test_cassandra_mod.py

```diff
@@ -0,0 +1,137 @@
+"""
+    :codeauthor: Rupesh Tare <rupesht@saltstack.com>
+
+    Test cases for salt.modules.cassandra_mod
+"""
+
+import pytest
+
+import salt.modules.cassandra_mod as cassandra
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {cassandra: {}}
+
+
+def test_compactionstats():
+    """
+    Test for Return compactionstats info
+    """
+    mock = MagicMock(return_value="A")
+    with patch.object(cassandra, "_nodetool", mock):
+        assert cassandra.compactionstats() == "A"
+
+
+def test_version():
+    """
+    Test for Return the cassandra version
+    """
+    mock = MagicMock(return_value="A")
+    with patch.object(cassandra, "_nodetool", mock):
+        assert cassandra.version() == "A"
+
+
+def test_netstats():
+    """
+    Test for Return netstats info
+    """
+    mock = MagicMock(return_value="A")
+    with patch.object(cassandra, "_nodetool", mock):
+        assert cassandra.netstats() == "A"
+
+
+def test_tpstats():
+    """
+    Test for Return tpstats info
+    """
+    mock = MagicMock(return_value="A")
+    with patch.object(cassandra, "_nodetool", mock):
+        assert cassandra.tpstats() == "A"
+
+
+def test_info():
+    """
+    Test for Return cassandra node info
+    """
+    mock = MagicMock(return_value="A")
+    with patch.object(cassandra, "_nodetool", mock):
+        assert cassandra.info() == "A"
+
+
+def test_ring():
+    """
+    Test for Return ring info
+    """
+    mock = MagicMock(return_value="A")
+    with patch.object(cassandra, "_nodetool", mock):
+        assert cassandra.ring() == "A"
+
+
+def test_keyspaces():
+    """
+    Test for Return existing keyspaces
+    """
+    mock_keyspaces = ["A", "B", "C", "D"]
+
+    class MockSystemManager:
+        def list_keyspaces(self):
+            return mock_keyspaces
+
+    @pytest.fixture
+    def configure_keyspaces():
```

Comment:
This fixture is not really being used, right?

Reply:
Removed, thanks.